report_xlsx_helper/readme/INSTALL.rst

```diff
@@ -0,0 +1 @@
+This module requires report_xlsx version 11.0.1.0.3 or higher.
```

Comment:
Is this state ment still valid?
I think it would need version 13.0.xxx isn't it?

Reply:
Thank for your review, Fixed.